beacon-chain/types/block.go

```diff
@@ -31,18 +31,7 @@ func NewBlock(data *pb.BeaconBlockResponse) (*Block, error) {
 		return nil, errors.New("invalid block data, parent hash should be 32 bytes")
 	}
 
-	return &Block{data}, nil
-}
-
-// NewGenesisBlock returns the canonical, genesis block for the beacon chain protocol.
-func NewGenesisBlock() (*Block, error) {
-	genesisTime := time.Date(2018, time.July, 21, 12, 0, 0, 0, time.UTC)
```

Comment:
Can you set this time to the current time?

Reply:
Was about to say, it was weird to have it be a static date. Will do 👍